Source/WebKit/PdfJSGResources.cmake

```diff
@@ -5,6 +5,7 @@ macro(WEBKIT_BUILD_PDFJS_GRESOURCES _derived_sources_dir)
     add_custom_command(
         OUTPUT ${_derived_sources_dir}/PdfJSGResourceBundle.xml
         DEPENDS ${TOOLS_DIR}/glib/generate-pdfjs-resource-manifest.py ${PdfJSFiles}
+        COMMAND ${CMAKE_COMMAND} -E rm -f ${_derived_sources_dir}/PdfJSGResourceBundle.deps
```

Comment:
This `rm` command support was added in CMake 3.17. Minimum required version ATM is 3.12, so this ends up failing on older versions of CMake.

Reply:
Drat

Reply:
I thought I was being clever and cross-platform friendly by doing this instead of normal `rm`. Alas. I'll change it to fall back to normal `rm` from PATH for older versions of CMake. That will fail on Windows, but (a) this file is currently only used by GTK and WPE, which don't work on Windows, and (b) people compiling for Windows can upgrade to newer CMake versions to avoid this.

Reply:
Turns out `cmake -E rm` is a new replacement for `cmake -E remove`. The `-E remove` is deprecated, but the behavior difference does not matter in this case, so I'll switch to that.

Reply:
https://bugs.webkit.org/show_bug.cgi?id=244728